assets/js/phoenix_live_view.js

```diff
@@ -56,6 +56,18 @@ const COMPONENTS = "c"
 let DEBUG = process.env.NODE_ENV !== 'production';
 let logError = (msg, obj) => console.error && console.error(msg, obj)
 
+function detectDuplicateIds() {
+  let ids = new Set()
+  let elems = document.querySelectorAll('*[id]')
+  for (let i = 0, len = elems.length; i < len; i++) {
+    if (!ids.has(elems[i].id)) {
```

Comment:
Hi @snewcomer 👋 

Just see this PR and I would like to suggest a small improvement. I think it's better if we  evaluate the positive case first instead of inverting the condition. Usually, it's better (IMO) for readability. At least it's trivial for my head when a read like that:

```
If (true) {
 do_positive_case()
else {
 do_negative_case()
}
```

Nice work 💪